### PR TITLE
Addition of New Spanish Translations

### PR DESCRIPTION
**Pull request by AlvaroMolinaCL**
I have added new translations, so I request the merger of these changes with the source code. Thanks.

### CODE REVIEW

File: i18n/miraheze/es.json [hunk 1]
```diff
@@ -2,30 +2,54 @@
...
+	"group-rollbacker-member": "reversor",
+	"grouppage-rollbacker": "m:Meta:Rollbacker",
+	"group-autopatrolled": "Autoverificados",
+	"group-autopatrolled-member": "autoverificado",
+	"grouppage-autopatrolled": "m:Meta:Autopatrolled",
+	"group-confirmed": "Confirmados",
+	"group-confirmed-member": "confirmado",
 	"grouppage-wikicreator": "m:Wiki creators",
-	"protect-level-user": "Permitir solo los usuarios registrados",
+	"protect-level-user": "Solo usuarios registrados",
```
Comment (Reception123) [inline i18n/miraheze/es.json]: The english version is "Allow only registered users" so please change it back to "Permitir"

File: i18n/miraheze/es.json [hunk 2]
```diff
@@ -2,30 +2,54 @@
...
 	"restriction-delete": "Borrar",
-	"restriction-level-user": "Permitir solo los usuarios registrados",
+	"restriction-level-user": "Solo usuarios registrados",
```
Comment (Reception123) [inline i18n/miraheze/es.json]: same as above

Comment (AlvaroMolinaCL) [inline i18n/miraheze/es.json]: Done.

File: i18n/miraheze/es.json (hunk 1 above)
Comment (AlvaroMolinaCL) [inline i18n/miraheze/es.json]: Done.

**Pull request merged**